Apromore-Custom-Plugins/Merge-Portal-Plugin/src/main/resources/META-INF/spring/osgi-context.xml

```diff
@@ -41,4 +41,11 @@
     <osgi:reference id="mergeService" interface="org.apromore.plugin.merge.logic.MergeService"/>
     <osgi:reference id="domainService" interface="org.apromore.service.DomainService"/>
 
+    <osgi:service interface="org.apromore.plugin.portal.WebContentService">
+        <beans:bean class="org.apromore.plugin.portal.BundleWebContentService">
+            <beans:constructor-arg ref="bundleContext"/>
+            <beans:property name="include" value="/merge/icon.svg"/>
+        </beans:bean>
+    </osgi:service>
+
```

Comment:
The icon resource location "/merge/icon.svg": can it be configured in one place instead of leaving two places: here and MergePlugin.getIconPath()?

Reply:
This is a (somewhat) unusual case where the ONLY web content for the merge plugin is its icon.  In the more general case, the icon path and the include regexp wouldn't match like this.